pliers/extractors/audio.py

```diff
@@ -618,3 +618,54 @@ def _extract(self, stim):
         return ExtractorResult(preds, stim, self, features=labels,
                                onsets=onsets, durations=[dur]*len(onsets),
                                orders=list(range(len(onsets))))
+
+
+class TimbrePitchExtractor(AudioExtractor):
+    ''' Extract Timbre and Pitch from Mel-Frequency Ceptral Coefficients.
+
+    Args:
+    n_mfcc (int): specifies the number of MFCC to extract
+    n_coefs(int): cepstrum coefficients to keep in low quefrency spectrum
+    hop_length (int): hop length in number of samples
+    librosa_kwargs (optional): Optional named arguments to pass to librosa
+    '''
+
+    _log_attributes = ('hop_length', 'n_coefs', 'n_mfcc')
+
+    def __init__(self, n_mfcc=48, n_coefs=13, hop_length=1024, **librosa_kwargs):
+        self.n_mfcc = n_mfcc
+        self.n_coefs = n_coefs
+        self.hop_length = hop_length
+        self.librosa_kwargs = librosa_kwargs
+        super().__init__()
+
+    def _extract(self, stim):
+
+        #https://librosa.org/doc/main/generated/librosa.feature.mfcc.html
+        mfccs = librosa.feature.mfcc(y=stim.data, 
```

Comment:
I think our only concern is that we already have an MFCC extractor and it "feels bad" to not re-use it properly.

@tyarkoni does it bother you to "re-use" MFCC here. We could also make this extractor more general way to extract groups of coefficients not just "high and low". I also don't want to over-think this though and rather just merge and iterate later if needed. 

FWIW, I think the approach of taking the top 13 components to extract high and low energy is a pretty common thing to do in audio feature extraction.

e.g.:

"The components of MFCCs are the first few DCT coefficients that describe the coarse spectral shape. The first DCT coefficient represents the average power in the spectrum. The second coefficient approximates the broad shape of the spectrum and is related to the spectral centroid. The higher-order coefficients represent finer spectral details (e.g., pitch). In practice, the first 8–13 MFCC coefficients are used to represent the shape of the spectrum. "
https://www.sciencedirect.com/topics/computer-science/cepstral-coefficient

Reply:
Fine with me to call it internally instead of invoking another extractor, seeing as it's just one call. I think it might even be preferable this way—we don't need all of the standard logging machinery to kick in.